gflownet/proxy/corners.py

```diff
@@ -30,6 +30,15 @@ def setup(self, env=None):
             self.cov_inv = torch.linalg.inv(cov)
             self.mulnormal_norm = 1.0 / ((2 * torch.pi) ** 2 * cov_det) ** 0.5
 
+    @property
+    def min(self):
+        if not hasattr(self, "_min"):
+            mode = self.mu * torch.ones(
+                self.n_dim, device=self.device, dtype=self.float
+            )
+            self._min = self(torch.unsqueeze(mode, 0))[0]
+        return self._min
```

Comment:
So if the caller of that property (accidently) does anything inplace on the returned tensor, that change will affect everyone else calling min in the future?
If so, it feels like it might lead to bugs that would be really difficult to track.
Same comment applies to the other proxies that implement the same strategy : torus (for min and norm) and uniform (for min).

Reply:
Uhm I see. Do you have a suggestion of a better way of doing this?

Reply:
Let's keep the conversation on this issue, please: https://github.com/alexhernandezgarcia/gflownet/issues/243